hibernate-reactive-core/src/test/java/org/hibernate/reactive/testing/TestingRegistryRule.java

```diff
@@ -71,10 +82,64 @@ public ServiceRegistry getParentServiceRegistry() {
             return null;
         }
 
-        @Override
+        @Override @SuppressWarnings("unchecked")
         public <R extends Service> R getService(Class<R> serviceRole) {
-            if ( serviceRole == VertxInstance.class )
+            if ( serviceRole == VertxInstance.class ) {
                 return (R) vertxService;
+            }
+            else if ( serviceRole == JdbcEnvironment.class ) {
+                return (R) new JdbcEnvironment() {
+                    @Override
+                    public Dialect getDialect() {
+                        return new MySQL8Dialect();
```

Comment:
Why do you need this change to TestingRegistryRule?

Reply:
To avoid an NPE, IIRC.

Comment it out; see what happens.🤷‍♂️